src/App.tsx

```diff
@@ -1,27 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const fiveFirstGoods = () => {
+    get5First()
+      .then((data) => {
+        setGoods(data);
+      });
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const allGoods = () => {
+    getAll()
+      .then((data) => {
+        setGoods(data);
+      });
+  };
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const redGoods = () => {
+    getRedGoods()
+      .then((data) => {
+        setGoods(data);
+      });
+  };
 
```

Comment:
DRY

Reply:
Sorry, but I didnt understand what do you mean in this case, corrected and submitted with all corrected but this one(